lib/crypto/packaged.py

```diff
@@ -27,7 +27,8 @@ class SigningError(Exception):
 def get_endpoint(file_obj):
     """Get the endpoint to sign the file, depending on its review status."""
     server = settings.SIGNING_SERVER
-    if file_obj.version.addon.status != amo.STATUS_PUBLIC:
+    if (file_obj.status == amo.STATUS_BETA or
+            file_obj.version.addon.status != amo.STATUS_PUBLIC):
```

Comment:
Scrap that, I closed #524. It would have been nice to have it (code quality wise), but it's too late for that, as we've already done most of the testing/QA